dotnet-roslyn/config/Audacia.CodeAnalysis.EntityFrameworkCore/Audacia.CodeAnalysis.EntityFrameworkCore.csproj

```diff
@@ -15,7 +15,7 @@
     </ItemGroup>
 
     <ItemGroup>
-        <PackageReference Include="Microsoft.EntityFrameworkCore.Analyzers" Version="6.0.0" />
+        <PackageReference Include="Microsoft.EntityFrameworkCore.Analyzers" Version="9.0.0" />
```

Comment:
Are there rules to review since upgrading this?

Reply:
Reverted as these are upgrading quite a few versions and not in the scope of this work